sourcery_analytics/cli/partials.py

```diff
@@ -99,6 +99,11 @@ def aggregate_csv_output(aggregation_method, method_metric, methods, metrics):
 
 
 def read_settings(settings_file: pathlib.Path, console: Console) -> Settings:
+    """Loads settings in the CLI.
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `read_settings`

Reason for update: The docstring was added to the function.

Suggested new docstring:
```
"""Loads settings in the CLI.

Wraps the basic settings loader in order to print relevant error messages and
exit with correct codes."""
```